workflow/scripts/preprocess/nd2_to_tif.py

```diff
@@ -0,0 +1,11 @@
+import snakemake
+from lib.preprocess.preprocess import nd2_to_tif
+from skimage.io import imsave
+
+# convert the ND2 file to a TIF image array
+image_array = nd2_to_tif(
+    snakemake.input[0], channel_order_flip=snakemake.params.channel_order_flip
+)
+
+# save TIF image array to the output path
+imsave(snakemake.output[0], image_array)
```

Comment:
I think before you merge this, it would be good to test if the things that you are saving are compatible with Fiji. I would download Fiji and Napari

https://imagej.net/software/fiji/downloads
https://napari.org/dev/tutorials/fundamentals/installation.html

And see what this tiff format is compatible with

We will also need to check if things work well with downstream steps given that we are massively changing how we are saving images. I am assuming that you can compare the analysis on the few tiles that you did in dev with these. I am a bit worried about this, but also understand the importance of streamlining this

Reply:
Makes sense. I'll check that these are compatible with Fiji/Napari. Definitely don't want to mess up the downstream steps.

Reply:
Sounds good

Reply:
Confirmed I am able to view these converted tifs. Merging!